persistence/src/main/scala/com/evolutiongaming/akkaeffect/persistence/PersistentActorOf.scala

```diff
@@ -16,19 +16,19 @@ import scala.concurrent.duration.*
 
 object PersistentActorOf {
 
+  // format: off
   /** Describes lifecycle of entity with regards to event sourcing & PersistentActor Lifecycle phases:
     *
-    *   1. RecoveryStarted: we have id in place and can decide whether we should continue with recovery 2. Recovering :
-    *      reading snapshot and replaying events 3. Receiving : receiving commands and potentially storing events &
-    *      snapshots 4. Termination : triggers all release hooks of allocated resources within previous phases
+    *   1. RecoveryStarted: we have id in place and can decide whether we should continue with recovery 
+    *   2. Recovering: reading snapshot and replaying events 
+    *   3. Receiving: receiving commands and potentially storing events & snapshots 
+    *   4. Termination: triggers all release hooks of allocated resources within previous phases
     *
-    * @tparam S
-    *   snapshot
-    * @tparam E
-    *   event
-    * @tparam C
-    *   command
+    * @tparam S snapshot
+    * @tparam E event
+    * @tparam C command
     */
+    // format: on
```

Comment:
let's not go this way - we do not strive for vertical compactness and for list we can use `-` easily - we never reference the lifecycle phases by their number 

same goes for `fmt` hacks in `EventSourcedActorOf`

Reply:
I strongly disagree: until scaladoc cannot properly format the docs (please check how it was _before_ my change) I will keep disabling it for the docs. 

Alternative approach is in disabling docs formatting at all:
```
docstrings.style = keep
```

Reply:
maybe https://scalameta.org/scalafmt/docs/configuration.html#docstringswrap:
```
docstrings.wrap = fold
```
though have to wait for 3.8.2...

Reply:
Numbered lists can work just fine, they just have to be formatted like this: https://contributors.scala-lang.org/t/do-numbered-lists-actually-work-in-scaladoc-markup/2748/2

Reply:
yep, trade-off with readability from source. Why should we prefer rendered version over source one?

Reply:
because it can be formatted by scalafmt? 😄 As a middle ground, I agree with the suggestion to use `-`

Reply:
> because it can be formatted by scalafmt?

my question was different: why should I sacrifice source over rendered version? GitHub does not render docs, VSCode does not render docs, i can bet that vim/emacs (if any) + metals does not render docs ... please consider others who does not use IntelliJ

> As a middle ground, I agree with the suggestion to use `-`

It has different meaning - not sequence but set of something without ordering. In this case ordering is preferable

Reply:
maybe something like:
```scala
/** Lifecycle
  *   - 1. create
  *   - 2. use
  *   - 3. kill
  */
```

Reply:
Nice, I like it & will use. Tnanks

Reply:
ok, let's wait for 3.8.2 release... and fix in Metals